dart/lib/src/scope.dart

```diff
@@ -81,6 +81,10 @@ class Scope {
 
   final SentryOptions _options;
 
+  final List<Attachment> _attachements = [];
+
+  List<Attachment> get attachements => UnmodifiableListView(_attachements);
```

Comment:
What's the benefit over `List.unmodifiable`? I'd stick to `List.unmodifiable` for now and raise an issue to use `UnmodifiableListView`, also for Map instead of `Map.unmodifiable` if they bring benefits

Reply:
Ist think `UnmodifiableListX` is probably the way to go: https://stackoverflow.com/a/61404767/12002462

Reply:
nice, open an issue to discuss and replace with `UnmodifiableListX` (milestone 6.0.0), but let's use the `List.unmodifiable` for consistency here and not get blocked by it, the reason is while the `view` list makes sense, it will be different from the other SDKs that just give a snapshot of the list at the given moment.